frontend/src/components/common/modal/Modal.module.scss

```diff
@@ -58,6 +58,6 @@ $title-height: $height-4;
   min-height: $title-height;
   text-align: center;
   @include mobile {
-    padding-right: 3rem;
+    padding-right: 3.5rem;
```

Comment:
Minor suggestion, I tried this with `padding: 0 2rem` to make it centered.

Reply:
I just tried `padding: 0 2rem`, but the close button still overlaps with the text at some widths.

Reply:
Ah okay, I didn't test across different widths, we can stick to `padding-right: 3.5rem;`then